### PR TITLE
Side Views of The binary tree

### PR DESCRIPTION
**Pull request by Simi06**
Fixed issue no. #7183 
Written in CPP, The code is well explained and optimized for minimum time and space complexity.

### CODE REVIEW

File: C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  	Node*root=new Node(1);
+  	root->left=new Node(2);
+  	root->right=new Node(3);
+  	root->left->left=new Node(4);
+  	root->left->right=new Node(5);	
+	root->right->left=new Node(6);
+  	root->right->right=new Node(7);
+  	root->left->left->right=new Node(8);
+  	root->right->right->left=new Node(9);
+  	root->right->right->left->left=new Node(10);
```
Comment (siddharth25pandey) [inline C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp]: Use Dynamic Input, also display messages in print form, what are you taking input from the user and what are you showing output to the user for better UI

File: C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        \
+        10
+        
+        here the root node is 1
+        the right side view of the tree would print 1 3 7 9 10
+        the left side view of the tree would print  1 2 4 8 10
+*/
+// including the header files
+ #include<iostream>
+ #include<queue>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp]: Use single header file **bits/stdc++**

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Simi06 Do the suggested changes

File: C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp (hunk 1 above)
Comment (Simi06) [reply C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp]: okay, sir.
On it

File: C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp (hunk 2 above)
Comment (Simi06) [reply C-Plus-Plus/ds/Side_Views_of_Binary_tree.cpp]: alright

**Comment by siddharth25pandey** [comment]
@Simi06 Also add the Update in README file

**Comment by Simi06** [comment]
@siddharth25pandey  Sir i have  worked on the changes you said, please review them once.

**Comment by siddharth25pandey** [comment]
> @Simi06 Also add the Update in README file

@Simi06 Not done yet

**Comment by Simi06** [comment]
@siddharth25pandey  Sir I have updated the README. Please check

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Code  alignment not proper
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add a new line at the EOF
- Readme.md missing

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Simi06 README file update is not added yet

**Comment by Simi06** [comment]
> * Code  alignment not proper
> * Add sample I/O at the end of the Code
> * Add Time and Space Complexity at the end of the Code
> * Add a new line at the EOF
> * Readme.md missing

@siddharth25pandey @ankitaggarwal23 please review the changes.

**Comment by siddharth25pandey** [comment]
> > * Code  alignment not proper
> > * Add sample I/O at the end of the Code
> > * Add Time and Space Complexity at the end of the Code
> > * Add a new line at the EOF
> > * Readme.md missing
> 
> @siddharth25pandey @ankitaggarwal23 please review the changes.

@Simi06 Please Check [this](https://github.com/TesseractCoding/NeoAlgo/pull/7319#pullrequestreview-671814227)

**Comment by Simi06** [comment]
@siddharth25pandey, am sorry sir but I edited the README and then committed the changes as well.
https://github.com/Simi06/NeoAlgo/commit/821ef5659a2fbc657214267e2e81c597851c08a7
could you please tell me where am I doing wrong?

**Comment by Simi06** [comment]
@siddharth25pandey  @ankitaggarwal23 please check now, am sorry for all the trouble.

**Review by siddharth25pandey — APPROVED** [review]
@Simi06 LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @Simi06 for her code

**Pull request merged**